src/metrics.jl

```diff
@@ -196,15 +196,15 @@ end
 
 """
     get_tradeoff_metrics(predicted_soft_labels, elected_hard_labels, class_index;
-                         thresholds, binarize=binarize_by_threshold)
+                         thresholds, binarize=binarize_by_threshold, class_labels=missing)
 
 Return [`TradeoffMetricsRow`] calculated for the given `class_index`, with the following
 fields guaranteed to be non-missing: `roc_curve`, `roc_auc`, pr_curve`,
 `reliability_calibration_curve`, `reliability_calibration_score`.` $(BINARIZE_NOTE)
 (`class_index`).
 """
 function get_tradeoff_metrics(predicted_soft_labels, elected_hard_labels, class_index;
-                              thresholds, binarize=binarize_by_threshold)
+                              thresholds, binarize=binarize_by_threshold, class_labels = missing)
```

Comment:
I guess we don't have a formatter setup here or it would complain:
```suggestion
                              thresholds, binarize=binarize_by_threshold, class_labels=missing)
```

Reply:
I ran a formatter so it should be fixed now :)